packages/algob/src/lib/validation-errors.ts

```diff
@@ -0,0 +1,26 @@
+import * as z from 'zod';
+import { ZodErrorCode, ZodSuberror } from "zod/lib/src/ZodError";
+
+function parseZodSubError (e: ZodSuberror, indent: number): string {
```

Comment:
Let's put this file next to the `errors-list.ts` file (in same for)

Reply:
Moved.